src/app/shared/lib/file-download.ts

```diff
@@ -0,0 +1,38 @@
+export function createAndDownloadFile(fileName: string, content: string) {
```

Comment:
```suggestion
export function downloadAsTextFile(fileName: string, content: string) {
```

Let's mention that it'll be a text file, to make things more obvious

Reply:
Thank you for the suggestion!